[Feature/功能]: Docker 镜像，使用 root 用户运行，避免 Claw Cloud 等挂载的磁盘没有写入权限，导致启动报错

### Pre-submission Checklist / 提交前检查

- [x] I have searched the [existing issues](https://github.com/looplj/axonhub/issues) and this feature has not been requested / 我已搜索现有 issues，此功能尚未被提出
- [x] I have read the [documentation](https://github.com/looplj/axonhub#readme) / 我已阅读文档

### Problem Statement / 问题描述

panic: sqlite: check foreign_keys pragma: reading schema information unable to open database file: out of memory (14)

### Proposed Solution / 期望方案

Docker 镜像，使用 root 用户运行，避免 Claw Cloud 等挂载的磁盘没有写入权限，导致启动报错

### Alternatives Considered / 备选方案

_No response_

### Feature Category / 功能分类

Other / 其他

### Additional Context / 其他补充信息

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -42,12 +42,10 @@ RUN --mount=type=cache,target=/go/pkg/mod \
 
 FROM alpine
 
-RUN apk add --no-cache ca-certificates tzdata && \
-    adduser -D -s /bin/sh axonhub
+RUN apk add --no-cache ca-certificates tzdata
 
 WORKDIR /app
 COPY --from=backend-builder /build/axonhub /app/axonhub
 
-USER axonhub
 EXPOSE 8090
 ENTRYPOINT ["/app/axonhub"]

